### PR TITLE
[AppLauncher] Fixes and improvements for homescreen and statusbar

### PR DESCRIPTION
**Pull request by eekkelund**
Add fontawesome icons to Launcher, fix minimized app rotation on AppSwitcher, add powersave options to statusbar.

### CODE REVIEW

File: src/qml/statusbar/BatteryPanel.qml [hunk 1]
```diff
@@ -32,16 +32,188 @@
...
+                    anchors.top:enablePowerSave.bottom
+                    anchors.topMargin: Theme.itemSpacingSmall
+
+                    property string entryPath: "system_settings/info/powersave/powersave_threshold"
+
+                    minimumValue: 1
+                    maximumValue: 99
+                    //label: "Battery threshold"
+                    showValue: true
+                  /*  style: SliderStyle{
```
Comment (neochapay) [inline src/qml/statusbar/BatteryPanel.qml]: Maybe remove ?

Comment (eekkelund) [inline src/qml/statusbar/BatteryPanel.qml]: Woops, thanks. Will do. :)

File: src/qml/SwitcherItem.qml [hunk 2]
```diff
@@ -83,15 +97,15 @@ MouseArea {
             script: switcherItemRoot.close()
         }
     }
-    Label {
+   /* Label {
```
Comment (neochapay) [inline src/qml/SwitcherItem.qml]: And here :)

Comment (eekkelund) [inline src/qml/SwitcherItem.qml]: Fixed!

**Pull request merged**